### ISSUE TITLE
RGB error

### ISSUE THREAD

**Issue by Jan20202020** [studioimaginaire/phue#177]
When i try to set an rgb value for a lamp i get this error:

ERROR, parameter, rgb, not avalible for light 2

My code:
`hue.set_light(2, {'rgb': (255, 102, 0)})`

**Comment by natcl** [comment]
Hue lights don’t support direct rgb control, you can only use hue or xy
mode.
Some libraries can help you do the conversion:
https://pypi.org/project/rgbxy/

--
lecaude.com
studioimaginaire.com

**Comment by Jan20202020** [comment]
And what about hex colors?
Are they working?

**Comment by natcl** [comment]
No, same principle, the hue API only supports XY and Hue/saturation modes,
you’ll need to find a way to convert rgb to the XY colorspace.
